lib/teleterm/api/proto/v1/service.proto

```diff
@@ -32,14 +33,34 @@ service TerminalService {
   rpc ListRootClusters(ListClustersRequest) returns (ListClustersResponse);
   // ListLeafClusters lists leaf clusters
   rpc ListLeafClusters(ListLeafClustersRequest) returns (ListClustersResponse);
-  // ListDatabases lists databases
-  rpc ListDatabases(ListDatabasesRequest) returns (ListDatabasesResponse);
+  // GetDatabases lists all databases without pagination
+  rpc GetAllDatabases(GetAllDatabasesRequest) returns (GetAllDatabasesResponse);
+  // GetDatabases returns a filtered and paginated list of databases
+  rpc GetDatabases(GetDatabasesRequest) returns (GetDatabasesResponse);
   // ListDatabaseUsers lists allowed users for the given database based on the role set.
   rpc ListDatabaseUsers(ListDatabaseUsersRequest) returns (ListDatabaseUsersResponse);
-  // ListServers lists servers
-  rpc ListServers(ListServersRequest) returns (ListServersResponse);
-  // ListKubes list kubes
-  rpc ListKubes(ListKubesRequest) returns (ListKubesResponse);
+  // GetAllServers returns all servers
+  rpc GetAllServers(GetAllServersRequest) returns (GetAllServersResponse);
+  // GetServers returns filtered, sorted, and paginated servers
+  rpc GetServers(GetServersRequest) returns (GetServersResponse);
+  // GetAccessRequests lists filtered AccessRequests
+  rpc GetAccessRequests(GetAccessRequestsRequest) returns (GetAccessRequestsResponse);
+  // GetAccessRequest retreives a single Access Request
+  rpc GetAccessRequest(GetAccessRequestRequest) returns (GetAccessRequestResponse);
+  // DeleteAccessRequest deletes the access request by id
+  rpc DeleteAccessRequest(DeleteAccessRequestRequest) returns (EmptyResponse);
+  // CreateAccessRequest creates an access request
+  rpc CreateAccessRequest(CreateAccessRequestRequest) returns (CreateAccessRequestResponse);
+  // ReviewAccessRequest submits a review for an Access Request
+  rpc ReviewAccessRequest(ReviewAccessRequestRequest) returns (ReviewAccessRequestResponse);
+  // GetRequestableRoles gets all requestable roles
+  rpc GetRequestableRoles(GetRequestableRolesRequest) returns (GetRequestableRolesResponse);
+  // AssumeRole assumes the role of the given access request
+  rpc AssumeRole(AssumeRoleRequest) returns (EmptyResponse);
+  // GetAllKubes list kubes
```

Comment:
The doc for `GetAllKubes` and `GetKubes` is identical. Do both RPCs do the same exact thing or is there a difference? If there's a difference, can we make it clear what the difference is?